### ISSUE TITLE
Changing page number for editions of a book causes results to scroll through all pages.

### ISSUE THREAD

**Issue by KhalidRmb** [internetarchive/openlibrary#1562]
As opposed to directly displaying the books in the requested page number. (As is displayed while browsing all the books by an author).

**Example**(Issue): https://openlibrary.org/works/OL15836721W/The_Great_Gatsby
**Should be like**: https://openlibrary.org/authors/OL27349A/F._Scott_Fitzgerald

**Comment by guyjeangilles** [comment]
@KhalidRmb Could you rephrase or post a screenshot of the issue? I'm not exactly understanding what the problem is.

Assigning jdlrobson (not tagging per request) per slack discussions as I think this issue is front-end related

**Comment by RayBB** [comment]
Closing for lack of clarity. Please reopen with the new issue template as needed.
